backend/src/main/java/com/huellapositiva/application/dto/VolunteerDto.java

```diff
@@ -11,4 +11,8 @@
 
     private final String id;
 
+    private final String emailAddress;
+
+    private final Boolean confirmed;
```

Comment:
Is this code used at all?

Reply:
The confirmed is used in FetchProposalAction to collect the value of whether that volunteer is confirmed or not.

Reply:
Ah ok, I searched for references and didn't saw anything, my bad.